ftl-build/x86_64-musl/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.10
+FROM alpine:edge
```

Comment:
I'd like this to be pinned to a release and not to a floating edge. Should that be fixed in this PR or after the fact?